website/_guides/doccomments.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: 'guide'
+title: 'Doc Comments'
+menuOrder: 3
+---
+
+# Document your code
+
+TypeDoc runs the TypeScript compiler and extracts type information from the generated compiler symbols.
+Therefore you don't have to include additional metadata within your comments, TypeScript specific elements
+like classes, enumerations or property types and access modifiers will be automatically detected.
+
+All comments are parsed as markdown. TypeDoc uses the [Marked](<https://github.com/chjj/marked>) markdown parser
+and [HighlightJS](<https://github.com/isagalaev/highlight.js>) to highlight code blocks within markdown sections.
+Additionally you can link to other classes, members or functions using double square brackets.
+
```

Comment:
are you going to add a note about [this](https://github.com/TypeStrong/typedoc/issues/917#issuecomment-448685993) as well here?

Reply:
I would like to. I wasn't sure if I wanted to do that in this PR or a follow up one. If I have time I'll add it.

Reply:
Cool!